src/pages/Product/Product.js

```diff
@@ -1,10 +1,35 @@
-import React from "react";
+import React, {useState} from "react";
 import "./Product.scss";
+import Count from "./components/Count/Count";
+import Review from "./components/Review/Review";
+import Color from "./components/Color/Color";
+import ColorButton from "./components/ColorButton/ColorButton";
 
 const Product = () => {
-  const price = 300;
+  const[count, setCount] = useState(1);
+  const[review, setreview] = useState(false);
+  const[color, setColor] = useState("white");
+  
+  const price = 300 * count;
   const totalPrice = price;
```

Comment:
```suggestion
  const price = 300;
  const totalPrice = price * count;
```

totalPrice라는 변수 이름에 맞게 위와 같이 구현하는 것이 더 자연스러워 보입니다.